oop/src/views/AccountTypePage.vue

```diff
@@ -357,4 +377,16 @@ export default {
     }
   }
 }
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+h2 {
+  font-size: 1.5rem;
+  font-weight: bold;
+}
```

Comment:
Is this `h2` rule needed? We should be keeping all `h1`, `h2`, etc sizes consistant. I think all `h2`s are bold in the Bootstrap CSS.